manageShips.php

```diff
@@ -0,0 +1,74 @@
+<?php
+require __DIR__.'/bootstrap.php';
+// Page to manage and display a ship table and link to view ships alone
+// for Edit/ Delete or Add ships
+
+use Service\Container;
+
+$container = new Container($configuration);
+
+$shipLoader = $container->getShipLoader();
+$ships = $shipLoader->getShips();
+?>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+           <meta http-equiv="X-UA-Compatible" content="IE=edge">
+           <meta name="viewport" content="width=device-width, initial-scale=1">
+           <title>Manage Ships</title>
+
+           <!-- Bootstrap -->
+           <link href="css/bootstrap.min.css" rel="stylesheet">
+           <link href="css/style.css" rel="stylesheet">
+           <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+
+           <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+           <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+           <!--[if lt IE 9]>
+             <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+             <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+           <![endif]-->
+    </head>
+
+    <!-- Error handling -->
+
+    <body>
+        <div class="container">
+            <div class="page-header">
+                <h1>Manage Our Ships</h1>
+            </div>
+            <table class="table table-hover">
+                <caption><i class="fa fa-rocket"></i> View/ Edit These Ships</caption>
+                <a href="index.php" class="btn btn-md btn-primary pull-right">Return Home</a>
+                <thead>
+                    <tr>
+                        <th>Ship</th>
+                        <th>Weapon Power</th>
+                        <th>Jedi Factor</th>
+                        <th>Strength</th>
+                        <th>Type</th>
+                    </tr>
```

Comment:
Add a blank `<th></th>` line here so the table data items all have a matching header. Currently, you have 5 columns in the header, and 6 in the body.

Reply:
Matched number of table headers to number of columns displayed